pkg/sriov-fec/nodeconfig.go

```diff
@@ -0,0 +1,348 @@
+package sriovfec
+
+import (
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/msg"
+	"github.com/openshift-kni/eco-goinfra/pkg/sriov-fec/sriovfectypes"
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	sriovfecnodeconfig = "SriovFecNodeConfig"
+)
+
+// Builder provides struct for the SriovFecNodeConfig object containing connection to
+// the cluster and the SriovFecNodeConfig definitions.
+type Builder struct {
+	Definition *sriovfectypes.SriovFecNodeConfig
+	Object     *sriovfectypes.SriovFecNodeConfig
+	apiClient  *clients.Settings
+	errorMsg   string
+}
+
+// AdditionalOptions additional options for sriovfecnodeconfig object.
+type AdditionalOptions func(builder *Builder) (*Builder, error)
+
+// NewBuilder creates a new instance of Builder.
+func NewBuilder(apiClient *clients.Settings, name, nsname string, label map[string]string) *Builder {
```

Comment:
Please re-name NewBuilder to NewNodeConfigBuilder.

Reply:
Re-named to NewNodeConfigBuilder.